src/app/shared/example-viewer/example-viewer.ts

```diff
@@ -217,18 +224,25 @@ export class ExampleViewer implements OnInit {
       const exampleBaseFileName = `${this.example}-example`;
       const docsContentPath = `/docs-content/examples-highlighted/${this.exampleData.packagePath}`;
 
+      const tsPath = normalizePath(`${exampleBaseFileName}.ts`);
+      const cssPath = normalizePath(`${exampleBaseFileName}.css`);
+      const htmlPath = normalizePath(`${exampleBaseFileName}.html`);
 
-      for (const fileName of this.exampleData.files) {
+      for (let fileName of this.exampleData.files) {
         // Since the additional files refer to the original file name, we need to transform
         // the file name to match the highlighted HTML file that displays the source.
         const fileSourceName = fileName.replace(fileExtensionRegex, '$1-$2.html');
         const importPath = `${docsContentPath}/${fileSourceName}`;
 
-        if (fileName === `${exampleBaseFileName}.ts`) {
+        // Normalize the path to allow for more consistent displaying in the tabs,
+        // and to make comparisons below more reliable.
+        fileName = normalizePath(fileName);
+
+        if (fileName === tsPath) {
           this.exampleTabs['TS'] = importPath;
-        } else if (fileName === `${exampleBaseFileName}.css`) {
+        } else if (fileName === cssPath) {
           this.exampleTabs['CSS'] = importPath;
-        } else if (fileName === `${exampleBaseFileName}.html`) {
+        } else if (fileName === htmlPath) {
```

Comment:
These lines starting with 227 are the only ones changed. The rest is from Prettier..